_api/logger.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: documentation
+title: Logger
+description: The Logger API lets you access the server's logging facility.
+type: server
+---
+
+The Logger API lets you access the server's logging facility.
+
+Messages can be logged at different severities, listed below and will be rendered to the same
```

Comment:
Messages can be logged at different severities (listed below) and will be rendered...